app/config.ts

```diff
@@ -0,0 +1,5 @@
+const commitmentContractAdress: string = process.env.COMMITMENT_CONTRACT_ADDRESS
+
+export {
```

Comment:
you can do `export const commitmentContractAdress: string = process.env.COMMITMENT_CONTRACT_ADDRESS`